src/Performance/Minibench.purs

```diff
@@ -88,4 +107,55 @@ bench
   :: forall eff a
    . (Unit -> a)
   -> Eff (console :: CONSOLE | eff) Unit
-bench = benchWith 1000
+bench = benchWith defaultNumSamples
+
+-- | Run an asynchronous function n times sequentially, calling the given
+-- | callback with the array of runtimes after all runs are completed.
+runAsync
+  :: forall e a
+   . Int
+  -> (Eff (console :: CONSOLE | e) Unit -> Eff (console :: CONSOLE | e) Unit)
+  -> (Array Number -> Eff (console :: CONSOLE | e) Unit)
+  -> Eff (console :: CONSOLE | e) Unit
+runAsync n f done = runAsync' n []
+  where
+    runAsync' 0 runs = done runs
+    runAsync' m runs = do
+      t1 <- runEffFn1 hrTime [0, 0]
+      f do
+        t2 <- runEffFn1 hrTime t1
+        void $ runAsync' (m - 1) ((fromHrTime t2):runs)
```

Comment:
I wonder if we should preallocate and use a mutable array here.

Reply:
Similar to what we were talking about with `ST`, I think it won't make a difference to the accuracy of the benchmark since it happens outside the `hrTime` measurements, and I think that kind of optimization only makes sense to do if it improves accuracy...